content/blog/minecraft-data-packs-scoreboards/index.md

```diff
@@ -0,0 +1,348 @@
+
+The data packs built in this series can be found in the [unicorn-utterances/mc-datapacks-tutorial](https://github.com/unicorn-utterances/mc-datapacks-tutorial/tree/main/3-scoreboards) repository. Feel free to use it for reference as you read through these articles!
+
+# Storing scores
+
+In many data packs, you might find a need to store information that can't be directly accessed through an entity or another command. A common way to do this is through the use of *scoreboards,* which can store a table of numbers for each entity or player. These can be used to reference player statistics, such as the number of blocks mined, or keep track of arbitrary values in your code.
+
+## Creating a scoreboard
+
+We can use the subcommands of `/scoreboard objectives` to create and modify any scoreboards that we use. Let's try making a scoreboard to track the number of animals that each player has spawned through our data pack.
+
+```shell
+scoreboard objectives add fennifith.animals_spawned dummy
+```
+
+This creates an objective named `fennifith.animals_spawned` that is connected to the `dummy` game statistic. We'll talk about what that means later on, but this effectively means the scoreboard will only be modified if you set its values yourself.
+
+## Scoreboard conventions
+
+### Namespaced objective names
+
+Players often want to have multiple data packs installed in their world at once. Since all scoreboards operate globally in the world, we need to make sure that our scoreboard names will not conflict with any scoreboards used by other data packs.
+
+To accomplish this, it is common to "namespace" your scoreboard names within your data pack by adding a certain prefix. Here, I've started my scoreboard names with `fennifith.animals` to indicate that they belong to my data pack.
+
+### Creating / removing Scoreboards
+
+Typically, all data packs will create any scoreboards they need in a `load.mcfunction` function, connected to the `#minecraft:load` function tag.
+
+Some data packs additionally create an `uninstall.mcfunction` file, not connected to any function tag, that can be executed to remove all of the data pack's scoreboard objectives. This is useful for when a player wants to remove your data pack from their world without leaving any of its behavior behind.
+
+## Setting values
+
+We can set values of a scoreboard using the `/scoreboard players` subcommands. Most of these subcommands accept two arguments for the `<targets>` and `<objective>` of the score to change. For example, the following command will set our entry in the `fennifith.animals_spawned` table to `1`.
+
+```shell
+#                     set our scoreboard entry
+#                     |   use the entry of the current player
+#                     |   |                    use "1" as the value to add
+#                     |   |                    |
+scoreboard objectives set @s fennifith.animals_spawned 1
```

Comment:
Double check all code snippets for errors between using `scoreboard players` and `scoreboard objectives`
Speaking of which, the difference needs to be explained between the two -because `scoreboard objectives add` is used later in "Tracking statistics"- as it may not be evident